src/Core/Translation/Export/TranslationCatalogueExporter.php

```diff
@@ -184,7 +184,8 @@ private function transformCatalogueMapToMessageCatalogue(Catalogue $catalogue, s
             $domainName = $domain->getDomainName();
             foreach ($domain->getMessages() as $message) {
                 $messageCatalogue->set($message->getKey(), $message->getTranslation(), $domainName);
-                $messageCatalogue->setMetadata($message->getKey(), ['file' => '', 'line' => ''], $domainName);
+                $metadata = $message->getMetadata();
```

Comment:
I was about to say update `TranslationCatalogueExporterTest` but it doesn't exist 😅

Reply:
So create `TranslationCatalogueExporterTest` :trollface: